components/Header.js

```diff
@@ -20,8 +20,7 @@ const StyledHeader = styled.header`
   text-align: center;
   font-family: var(--fontfamily-special);
   font-size: 1.3rem;
-  position: fixed;
-  top: 0;
+  height: 100%;
```

Comment:
Do you really need it? I don't see a difference without the `height: 100%`.

Reply:
deleted